src/index.html

```diff
@@ -1,15 +1,168 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST"
+    class="dataForm">
+      <fieldset class="inputGroup">
+        <legend>Personal information</legend>
+        <div class="inputBlock">
+          <label for="form_surname">Surname:</label>
+          <input
+          type="text"
+          id="form_surname"
+          name="surname"
+          autocomplete="off">
+        </div>
+        <div class="inputBlock">
+          <label for="form_name">Name:</label>
+          <input
+            type="text"
+            id="form_name"
+            name="name"
+            autocomplete="off"
+          >
+        </div>
+        <div class="inputBlock">
```

Comment:
classes for css are named in kebab-case
https://www.theserverside.com/definition/Kebab-case
```suggestion
        <div class="input-block">
```